lib/condor.py

```diff
@@ -188,3 +189,81 @@ def submit_dag(f, vargs, schedd_name, cmd_args=[]):
             print("Execution failed: ", e)
 
     return submit(subfile, vargs, schedd_name)
+
+
+class Job:
```

Comment:
Should the class be Job if we’re including clusters? Aren’t all jobs clusters, and not the reverse? What are your thoughts?

Reply:
I think we call it "jobsub-job-id" all over the place, even when we're referring to a cluster, so calling the class Job seems okay to me.